app/test_main.py

```diff
@@ -1 +1,50 @@
-# write your code here
+from app.main import can_access_google_page
+
+from unittest import TestCase, mock
+
+
+class TestCanAccesGooglePage(TestCase):
```

Comment:
There is a typo in the class name `TestCanAccesGooglePage`. It should be `TestCanAccessGooglePage`.